packages/cardhost/cardstack/seeds/templates.js

```diff
@@ -75,7 +75,7 @@ let eventCardTemplate = eventFactory.getDocumentFor(
         required: true,
         caption: 'Date & Time',
       }),
-      eventFactory.addResource('fields', 'image').withAttributes({
+      eventFactory.addResource('fields', 'background-image').withAttributes({
```

Comment:
This will need to be changed to use the new `decorative-image` core type after the other PR is merged, or if this PR is merged first, #1181 will need to be updated.